### ISSUE TITLE
Error: Cannot instantiate more than one Backbone.RelationalModel with the same id per type!

### ISSUE THREAD

**Issue by utiq** [PaulUithol/Backbone-relational#186]
Has anyone have this error? Error: Cannot instantiate more than one Backbone.RelationalModel with the same id per type!

**Comment by jameshowe** [comment]
I am experiencing a similar problem, partially touched on in this thread by @gfarrell.

I too am fetching a collection which contains `Backbone.RelationalModel`'s, the problem I have is when I re-fetch this collection the `relations` property does not appear to update (even though the returned data has now changed) because it's re-using the existing model from the store.

Shouldn't the library internally be calling `fetchRelated` per model in a collection when `fetch` is called? Anyway, just as a test, I tried clearing the cache before calling `fetch` i.e. `Backbone.RelationalModel.store.reset()` and that's when the error is thrown.

Can't seem to see a reasonable way around this, to be honest I've found my time using the library that I've spent more time working against it than I have with it :(
